web/application/views/airqo-node.php

```diff
@@ -436,5 +437,5 @@ function initMap() {
 
      // chart_d
 </script>
-<script src="https://maps.googleapis.com/maps/api/js?key=<API_KEY>&callback=initMap">
+<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAnK2dESOs8To3A01UdU3YpRNcC7cU_MKc&callback=initMap">
```

Comment:
please remove this API key from the repo. Try to explore other options of safely sharing this key with others in the development team. cc: @adongot @ja-odur